_data/team.yml

```diff
@@ -153,6 +153,18 @@
       url: "https://www.linkedin.com/in/ayushkhd"
     - name: flaticon-github-1
       url: "https://github.com/ayushkhd"
+      
+- name: Estee Cramer
+  role: PhD Student
+  description: |
+    Estee is a PhD student in epidemiology. She is interested in infectious disease transmission and malaria surveillance. She received her MPH in global health from Case Western Reserve University. In her free time she enjoys traveling.
+  image: /images/people/Estee.jpg
+    links:
```

Comment:
I think there is maybe an extra space in front of `links:`. Not sure it matters, but yaml is sometimes picky.